src/collection-preferences/content-display/__integ__/pages/content-display-page.ts

```diff
@@ -28,8 +28,13 @@ export default class ContentDisplayPageObject extends CollectionPreferencesPageO
     return this.wrapper.findModal().findContentDisplayPreference().findOptions();
   }
 
-  focusDragHandle(index = 0) {
-    return this.keys(new Array(5 + index * 2).fill('Tab'));
+  /**
+   * @param index The index of the drag handle to focus on. (Optional, default 0)
+   * @param offset The number of times to press the tab key to focus on the drag handle.
+   * Use 5 for collection preferences without searchable columns enabled, and 6 as the offset for searchable columns enabled. (Optional, default 5)
+   */
+  focusDragHandle(index = 0, offset = 5) {
```

Comment:
nit: Introducing the offset API exposes the detail on how this focusing is done and the consumer has to read the description before they can use it. I'd recommend having something like `{ hasSearchableColumns: boolean }` so that the offset choice can be done inside the method. In that case the added comments might not even be needed.

Alternatively, you can get the info of whether the search is enabled by querying the filter inside the method, like:
```
focusDragHanlde(index) {
  const isSearchable = !!this.wrapper.findModal().findContentDisplayPreference().findTextFilter():
  const offset = isSearchable ? 6 : 5;
  ...
}
```